CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.61.1](https://github.com/brave/brave-ios/releases/tag/v1.61.1)
+
+- Fixed Ad notification click event not triggering on iOS. ([#8619](https://github.com/brave/brave-ios/issues/8619))
```

Comment:
I don't think you need to mention `iOS` as these notes are all related to `iOS`. So probably can use:

```
- Fixed Brave Ad notification click event not being triggered. ([#8619](https://github.com/brave/brave-ios/issues/8619))
```

Added `Brave` in front of `ad` just so users don't get confused about that `ad` were talking about.